martin/src/pg/scripts/query_available_tables.sql

```diff
@@ -38,6 +39,7 @@ WITH
                f_geometry_column                    AS geom,
                srid,
                type,
+               -- 'geometry' AS column_type
```

Comment:
need?

Reply:
That's how we could distinguish between tables using a geometry vs geography column.
I initially thought we might need it, but the approach of simply casting everything to `::geometry` seems to be working well, so this can be removed unless we want to expose this information to the code for some other reason.